local-environment/dispatcher/conf/dispatcher/author.example.com.farm.any

```diff
@@ -0,0 +1,41 @@
+/author
+{
+  /clientheaders
+    {
+    "*"
+    }
+  /virtualhosts
+    {
+    "author.example.com"
```

Comment:
if some values need to be repeated consider regenerating it using expandable properties

`aem.props.expand()`

I mean, you are hardcoding port `4502` here but it should be in sync with declared port of `aem.config.instances` / through properties; as of GAP 6.0.1 they are now in sync when `aemUp` is being run by https://github.com/Cognifide/gradle-aem-plugin/blob/master/src/main/kotlin/com/cognifide/gradle/aem/instance/LocalInstance.kt#L263

Reply:
Those files are used to build the Docker image. GAP is not taking part in this process at all. I would rather keep those things loosely coupled. GAP is making sure services are up and helping to setup the environment. on the other hand, the developer is maintaining images he wants to use.